package/adapter.ts

```
@@ -61,11 +60,18 @@ export const getUsageAdapter = (context: AuthContext) => {
                     amount,
                     createdAt: new Date(Date.now()),
-                    beforeAmount: beforeAmount,
                     afterAmount: afterAmount
                 }
             })
             return usage
         },
+
+        syncUsage: async ({
+
+        }) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)